src/phaser/character/CharacterGroup.js

```diff
@@ -0,0 +1,36 @@
+class CharacterGroup {
+  constructor(scene) {
+    this._group = scene.add.group();
+  }
+
+  find(socketId) {
+    let character;
+
+    this._group.getChildren().forEach((groupCharacter) => {
+      if (socketId === groupCharacter.socketId) {
+        character = groupCharacter;
+      }
+    });
+
+    return character;
+  }
+
+  add(character) {
+    this._group.add(character);
+  }
+
+  remove(socketId) {
+    let result = false;
+
+    this._group.getChildren().forEach((groupCharacter) => {
+      if (socketId === groupCharacter.socketId) {
+        groupCharacter.destroy();
+        result = true;
+      }
+    });
+
+    return result;
+  }
```

Comment:
```suggestion
  remove(socketId) {
    let character;

    this._group.getChildren().forEach((groupCharacter) => {
      if (socketId === groupCharacter.socketId) {
        character = groupCharacter;
        groupCharacter.destroy();
      }
    });

    return character;
  }
```

`remove` 에서 삭제의 `return` 을 삭제 성공여부의 `result` 보다는 삭제한 `character` 을 반환할 수 있게 하는 건 어떨까요? 추후 로직에서 삭제한 캐릭터가 필요할수도 있을 것 같아서요. 삭제의 성공여부는 `find` 로직과 비슷하게 `undefined`을 리턴할때는 실패라고 인식하면 될 것 같아요.